onnxruntime/core/providers/qnn/builder/qnn_utils.cc

```diff
@@ -519,14 +522,26 @@ Status GetQuantParams(float rmin,
                       float rmax,
                       const Qnn_DataType_t qnn_data_type,
                       float& scale,
-                      int& zero_point) {
+                      int& zero_point,
```

Comment:
Since we're now quantizing int32 values, can we change the type of this zero_point parameter to `int32_t` to be explicit that it should be able to represent a 32-bit value?